Emby.Server.Implementations/HttpServer/HttpListenerHost.cs

```diff
@@ -69,6 +77,7 @@ public class HttpListenerHost : IHttpServer, IDisposable
 
             Instance = this;
             ResponseFilters = Array.Empty<Action<IRequest, HttpResponse, object>>();
+            GlobalResponse = localizationManager.GetLocalizedString("StartupEmbyServerIsLoading");
```

Comment:
This was removed from [ApplicationHost](https://github.com/jellyfin/jellyfin/pull/2601/files#diff-5ff7dd3f2f09dc8b526fc45b5264635fL782) and added here